apps/src/music/blockly/FieldChord.ts

```diff
@@ -77,10 +91,60 @@ export default class FieldChord extends Field {
   }
 
   protected render_() {
-    super.render_();
+    if (this.backgroundElement) {
+      this.backgroundElement.innerHTML = "";
+    }
+
+    const backingRectangle = GoogleBlockly.utils.dom.createSvgElement('rect',
+    {
+      'fill': '#54595e',
+      'x': 1,
+      'y': 1,
+      'width': FIELD_WIDTH,
+      'height': FIELD_HEIGHT
+    }, this.backgroundElement);
+
+    const graphNotes: ChordGraphNote[] = generateGraphDataFromChord({
+      chordEventValue: {
+        notes: this.getValue().notes,
+        playStyle: this.getValue().playStyle,
+        instrument: this.getValue().instrument
+      } as ChordEventValue,
```

Comment:
nit: can this just be `chordEventValue: {...this.getValue()}?` also guessing the cast to `as ChordEventValue` is necessary because `getValue()` returns `any`, right?

Reply:
Good call.  In fact, it looks like a simple `chordEventValue: this.getValue()` works!